developer-guides/rest-api/chat/ignoreuser/README.md

```diff
@@ -0,0 +1,37 @@
+# Ignore user
+
+Ignores an user in a chat.
+
+| URL | Requires Auth | HTTP Method |
+| :--- | :--- | :--- |
+| `/api/v1/chat.ignoreUser` | `yes` | `GET` |
+
+## Payload
+
+| Argument | Example | Required | Description |
+| :--- | :--- | :--- | :--- |
+| `rid` | `KLTM332QEwWLPJrjRPtcPHimPnLHsusiDF` | Required | The room ID. |
+| `userId` | `7aDSXtjMA3KPLxLjt` | Required | The User ID. |
+
+## Example Call
+
+```bash
+curl -H "X-Auth-Token: 9HqLlyZOugoStsXCUfD_0YdwnNnunAJF8V47U3QHXSq" \
+     -H "X-User-Id: aobEdbYhXfu5hkeqG" \
+     -H "Content-type:application/json" \
+     http://localhost:3000/api/v1/chat.ignoreUser?rid=KLTM332QEwWLPJrjRPtcPHimPnLHsusiDF&userId=7aDSXtjMA3KPLxLjt
+```
+
+## Example Result
+
+```json
+{
+    "success": true
+}
+```
+
+## Change Log
+
+| Version | Description |
+| :--- | :--- |
+|  | Added |
```

Comment:
The version that it has been add is `0.64.0`, as we can see [here](https://github.com/RocketChat/Rocket.Chat/pull/10517).